src/resources/SecurityCache/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './SecurityCache';
+export * from './SecurityCacheInterfaces';
```

Comment:
Did you add an export to _src/resources/index.ts_ ?

Reply:
:bowing_man: Thanks !